### DISCUSSION TITLE
is not null filter?

### DISCUSSION THREAD

**Discussion by begyy** [collerek/ormar#114] [Q&A] [answered]
#django 
Model.objects.filter(category__isnull=True)

ormar ?
![image](https://user-images.githubusercontent.com/44405438/108848553-f0fc2280-7602-11eb-93a9-964455558ded.png)

**Comment by collerek** [comment]
Issue #115 references this issue (actually it's already done and will be in next release with or filters)

**Comment by collerek** [comment, marked as answer]
Added in 0.9.7

**Comment by begyy** [comment]
thx

ср, 10 мар. 2021 г. в 10:23, collerek <notifications@github.com>:
